### PR TITLE
cluster: fill in reasons for scaling

### PR DESCRIPTION
**Pull request by xiang90**

### CODE REVIEW

File: pkg/spec/cluster.go [hunk 1]
```diff
@@ -187,17 +187,21 @@ func (cs *ClusterStatus) SetReason(r string) {
 	cs.Reason = r
 }
 
-func (cs *ClusterStatus) AppendScalingUpCondition() {
+func (cs *ClusterStatus) AppendScalingUpCondition(from, to int) {
+	reason := fmt.Sprintf("Current cluster size: %d, desired cluster size: %d", from, to)
```
Comment (hongchaodeng) [inline pkg/spec/cluster.go]: Make a common func "makeStatusReason"?

**Comment by xiang90** [comment]
@hongchaodeng Fixed.

**Comment by hongchaodeng** [comment]
lgtm

**Comment by hongchaodeng** [comment]
Manual merge.
Our jenkins GKE cluster isn't responsive.

**Pull request merged**